bokeh/server/tornado.py

```diff
@@ -69,6 +73,7 @@ def __init__(self, applications,
                     route = p[0]
                 else:
                     route = key + p[0]
+                route = self._prefix + route
```

Comment:
is it deliberate that we only prefix per_app_patterns and not toplevel_patterns?